candy-machine/program/src/processor/update.rs

```diff
@@ -27,8 +27,11 @@ pub fn handle_update_authority(
 ) -> Result<()> {
     let candy_machine = &mut ctx.accounts.candy_machine;
 
+    // Do not allow changing update authority if collections or freeze are active,
+    // unless the new authority is the same as the current authority.
     if let Some(new_auth) = new_authority {
-        if is_feature_active(&candy_machine.data.uuid, FREEZE_FEATURE_INDEX)
+        if (is_feature_active(&candy_machine.data.uuid, FREEZE_FEATURE_INDEX)
```

Comment:
Can you explain why we can't change authority when `FREEZE_FEATURE_INDEX` is active?  @samuelvanderwaal explained to me why its a problem for `COLLECTIONS_FEATURE_INDEX` but not seeing why its an issue for freeze.

Reply:
Now that I'm looking at it again with fresh eyes I'm not actually sure. I may have been confusing things in my head when I was going over this with Sam last night.